common/src/main/java/com/wynntils/screens/itemfilter/widgets/BooleanValueWidget.java

```diff
@@ -19,8 +21,11 @@ public class BooleanValueWidget extends GeneralValueWidget {
 
     private ConfirmedBoolean state = ConfirmedBoolean.UNCONFIRMED;
 
-    public BooleanValueWidget(String query, ItemFilterScreen filterScreen) {
-        super(Component.literal("Boolean Value Widget"), filterScreen);
+    public BooleanValueWidget(
+            List<StatProviderAndFilterPair> filters,
```

Comment:
`filters` doesn't seem to be used for any of the widgets, is it necessary?

Reply:
Oh yeah, forgot to remove those.

Reply:
Ok, I see the List widget uses it, which then makes the list passed into `onFiltersChanged` unused. Maybe it could be set there